### PR TITLE
[conv3d_transpose] Fix dim check for bias

### PR DESCRIPTION
**Pull request by peterjc123**
Per discussion with @thaink in https://github.com/tensorflow/tensorflow/commit/38a77f4bc1c897c75560a0beb53426f20b883f6e#r68775543 , the previous way to do the dim check for bias is not correct. So, we need this change.

### CODE REVIEW

**Review by thaink — APPROVED** [review]

**Comment by thaink** [comment]
I think you need to change MismatchBiasSizeTest in the test as well.

**Comment by peterjc123** [comment]
@thaink Done.

**Review by thaink — APPROVED** [review]

**Comment by peterjc123** [comment]
@thaink Is there any difficulty in landing this PR?

**Comment by thaink** [comment]
I am not sure why the recent update was not reflected in our internal PR. Let me try again.

**Review by roniesha1994 — COMMENTED** [review]
Griow
